naming/src/main/java/com/alibaba/nacos/naming/misc/HttpClientManager.java

```diff
@@ -131,16 +147,58 @@ protected Logger assignLogger() {
     }
     
     private static class ApacheSyncHttpClientFactory extends AbstractApacheHttpClientFactory {
-    
+        
         @Override
         protected HttpClientConfig buildHttpClientConfig() {
-            return HttpClientConfig.builder()
-                    .setConnectionTimeToLive(500, TimeUnit.MILLISECONDS)
+            return HttpClientConfig.builder().setConnectionTimeToLive(500, TimeUnit.MILLISECONDS)
                     .setMaxConnTotal(Runtime.getRuntime().availableProcessors() * 2)
-                    .setMaxConnPerRoute(Runtime.getRuntime().availableProcessors())
-                    .setMaxRedirects(0).build();
+                    .setMaxConnPerRoute(Runtime.getRuntime().availableProcessors()).setMaxRedirects(0).build();
+        }
+        
+        @Override
+        protected Logger assignLogger() {
+            return SRV_LOG;
         }
+    }
     
+    private static class ProcessorHttpClientFactory extends AbstractHttpClientFactory {
+        
+        private static final int CONNECTION_REQUEST_TIMEOUT = EnvUtil
```

Comment:
Static final variable get from properties? Static final means that it's a constants. Should not be configed.